### PR TITLE
N32: Notan v0.2.1

### PR DESCRIPTION
**Pull request by Nazariglez**
Part of #979

### CODE REVIEW

File: content/news/032/index.md [hunk 1]
```diff
@@ -232,6 +232,26 @@ March's full weekly devlogs: "This Week In Veloren...":
 
 ## Engine Updates
 
+### [Notan v0.2.1]
+
+![notan](notan.jpg)
+
+[Notan] is a simple and portable layer designed to create your own multimedia
+apps on top of it without worry about platform-specific code.
```
Comment (17cupsofcoffee) [inline content/news/032/index.md:240]:
```suggestion
apps on top of it without worrying about platform-specific code.
```

File: content/news/032/index.md [hunk 2]
```diff
@@ -232,6 +232,26 @@ March's full weekly devlogs: "This Week In Veloren...":
...
+
+The main goal is to provide a set of APIs and tools that can be used to create
+your project in an ergonomic manner without enforcing any structure or pattern,
+always trying to stay out of your way. The idea is that you can use it as a
+foundation layer or backend for your next app, game engine, or game.
+
+The version [v0.2.1] improves and adds features focusing on the creation of apps,
```
Comment (17cupsofcoffee) [inline content/news/032/index.md:247]:
```suggestion
Version [v0.2.1] improves and adds features focusing on the creation of apps,
```

**Review by 17cupsofcoffee — APPROVED** [review]

File: content/news/032/index.md [hunk 3]
```diff
@@ -232,6 +232,26 @@ March's full weekly devlogs: "This Week In Veloren...":
 
 ## Engine Updates
 
+### [Notan v0.2.1]
```
Comment (17cupsofcoffee) [inline content/news/032/index.md:235]:
```suggestion
### [Notan v0.2.1][Notan]
```

**Comment by Nazariglez** [comment]
Thanks for the fixes @17cupsofcoffee!

**Pull request merged**